packages/grid_client/src/primitives/nodes.ts

```diff
@@ -373,6 +377,101 @@ class Nodes {
       });
   }
 
+  /**
+   * Finds the index of a disk with a size greater than or equal to the specified disk size in the DisksPool.
+   *
+   * @param diskPools - An array of disk sizes available in the pool.
+   * @param disk - The required disk size to fit into the pool.
+   * @returns {boolean} - True if the disk can be fitted into the pool, otherwise false.
+   */
+  findDiskIndex(diskPools: number[], disk: number): boolean {
+    for (const index in diskPools) {
```

Comment:
why are we still looping with the index? you can use `of` instead of `in` to get the value

Reply:
to be able to edit the original disk pool value